packages/core/src/BaseRouter.tsx

```diff
@@ -78,6 +78,10 @@ const BaseRouter = {
   shouldActionChangeFocus(action: CommonAction) {
     return action.type === 'NAVIGATE';
   },
+
+  canGoBack() {
+    return false;
```

Comment:
yes, but I'll still leave it by default

Reply:
what I'm afraid of is that by leaving it here it's easy for routers to forget to implement it. since it's quite crucial to proper back button handling, maybe be it's better to force routers to implement it?